### ISSUE TITLE
Fix CSS import errors when Perses components are embedded

### ISSUE THREAD

**Issue by sjcobb** [perses/perses#894]
Recently, we moved a lot of our MUI theme into the @perses-dev/components package to make extending types easier, but this causes new issues when embedding Perses in apps built with certain frameworks like Next.js.

The newest versions of Next.js have [a workaround](https://github.com/vercel/next.js/discussions/27953#discussioncomment-3978605) for this, but it is bad practice for libraries like our's to import CSS files this way (more details: https://nextjs.org/docs/messages/css-npm)

These [Lato imports ](https://github.com/perses/perses/blob/main/ui/components/src/theme/typography.ts#L14-L17)specifically are causing problems.

## Screenshot

<img width="877" alt="image" src="https://user-images.githubusercontent.com/9369625/209268497-1eb48e79-b2d0-44bc-b976-853ebefc04ef.png">

**Comment by sjcobb** [comment]
Note: Errors are also being thrown related to lodash-es
```
error - Error [ERR_REQUIRE_ESM]: require() of ES Module /sjcobb/perses-embed-example/node_modules/lodash-es/lodash.js from /sjcobb/perses-embed-example/node_modules/@perses-dev/core/dist/cjs/utils/memo.js not supported.
Instead change the require of lodash.js in /sjcobb/perses-embed-example/node_modules/@perses-dev/core/dist/cjs/utils/memo.js to a dynamic import() which is available in all CommonJS modules.
```

**Comment by juliepagano** [comment]
In general, I don't know that we've been especially good about making sure Perses is SSR-friendly. Do you think that's something we should spend a little time investing in going forward?

**Comment by sjcobb** [comment]
> In general, I don't know that we've been especially good about making sure Perses is SSR-friendly. Do you think that's something we should spend a little time investing in going forward?

Yeah definitely, at least a spike to see where the gaps are and how much work it’ll be (do you have some ideas already?)
